lib/vanity/adapters/redis_adapter.rb

```diff
@@ -217,9 +217,13 @@ def ab_set_outcome(experiment, alternative = 0)
       end
 
       def destroy_experiment(experiment)
-        @experiments.del "#{experiment}:outcome", "#{experiment}:created_at", "#{experiment}:completed_at"
-        alternatives = @experiments.keys("#{experiment}:alts:*")
-        @experiments.del(*alternatives) unless alternatives.empty?
```

Comment:
@urbanautomaton just to make sure I understand, prefixes like "#{experiment}:participants:*" were missing from this list, correct?

Reply:
Yep, exactly. 👍

Reply:
(and in some cases that would've been a huge number of keys, hence the decision to use SCAN rather than KEYS then DEL)

Reply:
Looks like this wasn't atomic before, but if it's an issue, could probably come up with a lua script alternative.

Reply:
Yeah - none of the other adapters do this atomically, so I think it's probably okay unless we hear otherwise.